pkg/cloudprovider/providers/azure/vhd.go

```diff
@@ -1,38 +0,0 @@
-/*
```

Comment:
keep this file, this is how to create a blank vhd (unless it is already supported by azure).